test/common/filesystem/directory_test.cc

```diff
@@ -15,6 +15,17 @@
 namespace Envoy {
 namespace Filesystem {
 
+// NOLINTNEXTLINE(readability-identifier-naming)
+void PrintTo(const DirectoryEntry& entry, std::ostream* os) {
```

Comment:
sorry if I'm missing something obvious but why NOLINTNEXTLINE? Can you not just have Envoy standard naming?